content/2017/03/06/pattern-making.rst

```diff
@@ -0,0 +1,518 @@
+public: yes
+author: miriam
+tags: [Tools, Color, Typography, 'Style Guides', Accoutrement, Herman, OddSite, 'Open Design']
+image:
+  - src: '2017/pattern-making/NYC-sign.jpg'
+summary: |
+  Living Style Guide documentation on the web
+  is a difficult problem,
+  gaining a lot of attention in the last few years.
+  Let's take an in-depth look
+  at one way to store patterns directly in Sass,
+  and generate documentation automatically.
+
+
+Code Patterns & Style Guides
+============================
+
+I've often been asked
+how we sell style guides to our clients.
+The short answer is: *we don't*.
+Our clients want to put usable features
+in front of real users
+on a tight schedule.
+Style guides aren't the product,
+they're part of the process.
+
+Style guides are necessary,
+but keeping them alive
+can be difficult and time-consuming.
+We have to find ways
+to integrate documentation into the daily workflow,
+or it will get lost in the scramble
+for bigger and better features.
+
+As `Carl`_ likes to say:
+*un-tested code is legacy code –
+even if you wrote it an hour ago*.
+
+.. _Carl: /authors/carl/
```

Comment:
This isn't specific to this post, but I often wonder how we decide when to link to `/authors/carl/` vs `/birds/#bird-carl`?